packages/webapp/src/components/ErrorCircle.tsx

```diff
@@ -0,0 +1,15 @@
+import { XMarkIcon, ArrowPathIcon, LockClosedIcon } from '@heroicons/react/24/outline';
```

Comment:
Nb: we should only use tabler icons, maybe the figma was old

Reply:
@bodinsamuel Fixed in 946d9e36. I went ahead and changed the x icon to use ! instead, because that felt less like some sort of close button to me.